src/tests/ftest/soak/soak.py

```diff
@@ -474,20 +534,19 @@ def tearDown(self):
         """Define tearDown and clear any left over jobs in squeue."""
         print("<<tearDown Started>> at {}".format(time.ctime()))
         # clear out any jobs in squeue;
-        try:
-            if len(self.failed_job_id_list) > 0:
-                print("<<Cancel jobs in queue with ids {} >>".format(
-                    self.failed_job_id_list))
-                for job_id in self.failed_job_id_list:
-                    slurm_utils.cancel_jobs(job_id)
-        finally:
-            # One last attempt to copy any logfiles from client nodes
-            status = self.remote_copy(
-                self.node_list, self.rem_pass_dir, self.outputsoakdir)
-            if not status:
-                self.log.info(
-                    "Some logfiles may not be available from client node")
-            super(Soak, self).tearDown()
+
+        if len(self.failed_job_id_list) > 0:
+            print("<<Cancel jobs in queue with ids {} >>".format(
+                self.failed_job_id_list))
+            for job_id in self.failed_job_id_list:
+                slurm_utils.cancel_jobs(job_id)
+        # One last attempt to copy any logfiles from client nodes
+        status = self.remote_copy(
+            self.node_list, self.rem_pass_dir, self.outputsoakdir)
+        if not status:
+            self.log.info(
+                "Some logfiles may not be available from client node")
+        super(Soak, self).tearDown()
```

Comment:
Report the possible failed slurm job cancellations after attempting the inherited tear down steps:
```suggestion
        super(Soak, self).tearDown()
        if errors_detected:
            self.fail("Errors detected canceling slurm jobs")
```